fastlane/SnapshotHelper.swift

```diff
@@ -166,7 +166,6 @@ open class Snapshot: NSObject {
     }
 
     class func pathPrefix() throws -> URL? {
-        let homeDir: URL
```

Comment:
As you said elsewhere we didn't write this file... is it even being built?  I don't think we use it do we?  Maybe we could delete it instead?

Reply:
It is, yep - it's used for AtbUITests only though.